pom.xml

```diff
@@ -57,6 +57,16 @@
             <artifactId>java-client</artifactId>
             <version>${java-client.version}</version>
         </dependency>
+        <dependency>
```

Comment:
These dependencies come from my local .m2, so they include those changes waiting in this PR https://github.com/hartwigmedical/platinum-descriptor-language/pull/5